day2/exercise1/solution.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	words := [5]string {"quick","brown","fox","lazy","dog"}
+	var state = make(map[string]int)
+
+	charChannel := make(chan string)
+
+	go func() {
```

Comment:
write this function separately so that the function can be reusable. modular code.